.gitignore

```diff
@@ -0,0 +1,61 @@
+#ignore idea configuration
```

Comment:
It seems most of entries are redundant, could you please squash it?

Reply:
I have squashed it in commit [[Bulk Load: add tikv bulk load using spark](https://github.com/tikv/migration/pull/1/commits/7eb80947ac778e6ad6e4dbe3bb86836a08f27050)]